test/likelihoods/test_multitask_gaussian_likelihood.py

```diff
@@ -28,6 +28,27 @@ def _create_targets(self, batch_shape=torch.Size([])):
     def create_likelihood(self):
         return MultitaskGaussianLikelihood(num_tasks=4, rank=2)
 
+    def test_setters(self):
```

Comment:
maybe test that the proper errors are being raised here if you try to do sth incompatible?